docs/public-networks/concepts/transactions/pool.md

```diff
@@ -62,9 +53,24 @@ If you previously configured transaction pool behavior, upgrade to the layered t
 You can opt out of the layered transaction pool implementation by setting the
 [`--tx-pool`](../../reference/cli/options.md#tx-pool) option to `sequenced`.
 
-## Dropping transactions when the transaction pool is full
+## Sequenced transaction pool
+
+In the sequenced transaction pool, transactions are processed strictly in the order they are received.
+Although sequenced transaction pools lack the flexibility of layered pools, they help maintain a 
+consistent and transparent transaction order, which is often useful in private blockchains. 
+
+You can select the sequenced transaction pool by setting [`--tx-pool=sequenced`](../../reference/cli/options.md#tx-pool).
+
+If you set the enterprise configuration profile using [`--profile=enterprise`](../../how-to/use-configuration-file/profile.md#enterpriseprivate-profile) or [`--profile=private`](../../how-to/use-configuration-file/profile.md#enterpriseprivate-profile), the `sequenced` transaction pool is set by default.
 
-When the transaction pool is full, it accepts and retains local transactions in preference to remote transactions. If the transaction pool is full of local transactions, Besu drops the oldest local transactions first. That is, a full transaction pool continues to accept new local transactions by first dropping remote transactions and then by dropping the oldest local transactions.
+The sequenced transaction pool suits enterprise environments because it functions like a first-in-first-out (FIFO) queue and processes transactions in the order of submission, regardless of the sender. 
```

Comment:
I believe the following statements are misleading:

> In the sequenced transaction pool, transactions are processed **strictly in the order they are received**.
> Although sequenced transaction pools lack the flexibility of layered pools, they help maintain a **consistent and transparent transaction order**, which is often useful in private blockchains. 

> The sequenced transaction pool suits enterprise environments because it functions like a **first-in-first-out (FIFO) queue** and **processes transactions in the order of submission**, regardless of the sender. 

As one can believe that transactions are no longer ordered by gas prices, but solely on submission timestamp.

Reply:
@matthew1001 I think this comment is correct, since we usually linked the sequenced pool with gasless networks, then only the submission order is relevant, but in case there is a gas market, the order should be: gas price first, then submission order, do you agree?

Reply:
Yes, I believe that the order does take into account gas price - it's just that it's very rare people combine the two. I think the docs update can be a simple one, to qualify the current text. E.g. `In the sequenced transaction pool, if the network's gas price is 0, transactions are processed strictly in the order they are received...`. 

Then perhaps we need another paragraph e.g.

```
It is possible to configure a private blockchain with non-zero gas price. In this case the sequenced transaction pool does factor transaction gas price into the selection priority order.
```